config/schema/format_strawberryfield.schema.yml

```diff
@@ -614,3 +614,14 @@ field.formatter.settings.strawberry_citation_formatter:
         label: 'style'
     localekey:
       type: string
+condition.plugin.ado_type_condition:
+  type: condition.plugin
+  label: 'ADO Type Condition'
+  mapping:
+    ado_types:
+      type: array
```

Comment:
Hey, almost! this is like this (there is no array type)

 type: sequence
      sequence:
        type: string

Reply:
Ok, updated.